cpu/cortexm_common/Makefile.features

```diff
@@ -18,26 +18,38 @@ ifneq (,$(filter $(CPU_CORE),cortex-m4f cortex-m7))
 endif
 
 # Set CPU_ARCH depending on the CPU_CORE
+#
+# RUST_TARGET is only used when building Rust code; any users need to require
+# the `rust_target` feature to esnure things are checked properly.
 ifeq ($(CPU_CORE),cortex-m0)
   CPU_ARCH := armv6m
+  RUST_TARGET = thumbv6m-none-eabi
 else ifeq ($(CPU_CORE),cortex-m0plus)
   CPU_ARCH := armv6m
+  RUST_TARGET = thumbv6m-none-eabi
 else ifeq ($(CPU_CORE),cortex-m23)
   CPU_ARCH := armv8m
 else ifeq ($(CPU_CORE),cortex-m3)
   CPU_ARCH := armv7m
+  RUST_TARGET = thumbv7m-none-eabi
 else ifeq ($(CPU_CORE),cortex-m33)
   CPU_ARCH := armv8m
```

Comment:
Is there no `thumbv8m-none-eabi`?

Reply:
There probably is, but I haven't verified it yet. As with other architectures, I'm trying to start a bit slowly here (but if anyone has a board around where they can test the build results, I'm happy to open it up further).

Reply:
I can test on a `saml10-xpro` which has a Cortex-M23

Reply:
I'ved added them now (and rebuilt Makefile.ci with consideration for them), but left them commented out because the build servers will need [another update round](https://github.com/RIOT-OS/riotdocker/pull/164) for them. Things work up to firmware building (which is generally a good sign, as the linker usually refuses to work if the triples don't match), and you can test it by removing the comment mark, but I'd leave them out of this PR to not make it depend on the manual CI upgrades that can only start once the above-mentioned PR is merged.